Makefile

```diff
@@ -1,5 +1,5 @@
 __SIM_ID=`xcrun simctl list|egrep -m 1 '$(SIM_NAME) \([^(]*\) \([^(]*\)$$'|sed -e 's/.* (\(.*\)) (.*)/\1/'`
```

Comment:
Potentially unrelated, but this expression looks way complicated. I'd vote for keeping it, unless we can think of an way easier way of expressing the same thing....